pages/mdanalysis-as-a-building-block.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: MDAnalysis as a building block
```

Comment:
This title shows up in the menu bar. It is quite long. I'd like something shorter, along the lines of
- Who uses MDAnalysis?
- Made with MDA
- Uses

I don't particularly like having "MDAnalysis" in the title.

Do you have some better ideas?

Reply:
maybe "Used By"?